packages/shared/src/components/modals/PostModal.tsx

```diff
@@ -22,6 +22,7 @@ export function PostModal({
   isFetchingNextPage,
   ...props
 }: PostModalProps): ReactElement {
+  const [currentPage, setCurrentPage] = useState('');
```

Comment:
It's more accurate this way, rather than using empty string

```suggestion
  const [currentPage, setCurrentPage] = useState<string>();
```